helpers/config.py

```diff
@@ -49,6 +49,10 @@ def verify_settings():
         data['general'].get('eq_log_dir', ''),
         ''
         )
+    data['general']['char_name'] = get_setting(
```

Comment:
I don't think we need to actually expose this

Reply:
can avoid publishing to .json if I can figure a way to communicate the data out of LogReader object (which learns it from the filename), and into DeathLoopVaccine (which needs it for the regular expression for windowed tells).

I think the loc sharing code also (optionally) uses it.

Maybe just a global variable somewhere would be better than into the .json config file.

Reply:
took it out of .json file and moved it to a global variable

Reply:
Yeah, that's what I was suggesting.